client/src/app/layout/SidebarApp/SidebarApp.tsx

```diff
@@ -226,7 +226,15 @@ export const SidebarApp: React.FC = () => {
                     </NavLink>
                   </NavItem>
                 </NavExpandable>
-              ) : null}
+              ) : null}{" "}
```

Comment:
Looking great! A few small things but this is great progress. Is this space needed?